dashboard/src/app/utils/variables.ts

```diff
@@ -16,6 +16,13 @@ export const getVariableValue = (variable?: VariableValue) => {
 
   if (variable.bytes) {
     return '[bytes]'
+  } else if (variable.str) {
+    try {
+      const json = JSON.parse(variable.str)
+      return JSON.stringify(json, null, 4)
+    } catch {
+      return variable[key]
+    }
```

Comment:
What about `variable.jsonObj` and `variable.jsonArr`?

Reply:
Good catch